File: LogicBuildingProblems/divBy13.cpp
#include<iostream>
using namespace std;
bool divBy13(string str){
    int rem{};
    for(auto ch : str){
        rem = (rem * 10 + (ch - '0')) % 13;
    }
    return rem == 0;
}
int main(){
    string str{"2911285"};
    cout << (divBy13(str) ? "Yes" : "No");
}

File: LogicBuildingProblems/divBy13.go
package main

import "fmt"

func divBy13(str string) bool {
	rem := 0
	for i := 0; i < len(str); i++ {
		rem = (rem*10 + int(str[i]-'0')) % 13
	}
	fmt.Println(rem)
	return rem == 0
}
func main() {
	str := "2911285"
	fmt.Println(divBy13(str))

}
